messages/common.md

```diff
@@ -2,6 +2,6 @@
 
 Emit additional command output to stdout.
 
-# surveyRequestMessage
+# deprecationMessage
 
-We're continually improving Salesforce Code Analyzer. Tell us what you think! Give feedback at http://sfdc.co/CodeAnalyzerFeedback
+We will end support for v4.x of Code Analyzer in the upcoming months. We recommend users migrate to v5.x, which is currently in Beta and will be in GA soon. More information: https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/version-5.html
```

Comment:
```suggestion
We plan to stop supporting v4.x of Code Analyzer in the coming months. We highly recommend that you start using v5.x, which is currently in Beta. For information on v5.x, see https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/version-5.html.
```

Reply:
@jag-j , a few notes about my edits:

* Rather than "migrate", which to me implies there's an actual migration path/process/tool, I say "start using" because that's what users do, right?  They just install and use v5?  
* I removed the "will be GA soon" because I'm hesitant to promise anything like that.  Who knows, sometihng could happen and it takes a year to GA!   I assume that this warning will start once the product is officially Beta, right?